guide/03-the-gis/accessing-and-creating-content.ipynb

```diff
@@ -1492,7 +1492,8 @@
     "                'tags':'arcgis, python, earthquake, natural disaster, emergency'}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    earthquake_csv_item = root_folder.add(item_properties=csv_properties, data=csv_path,
</pre><p>the folder.add method returns a future, not the item itself, so need to do the whole job/result() thing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>